tests/Proto.Persistence.Tests/InMemoryProvider.cs

```diff
@@ -47,7 +49,9 @@ public Task<long> PersistEventAsync(string actorName, long index, object @event)
 
             events.Add(index, @event);
 
-            return Task.FromResult(0L);
+            long max = events.Max(x => x.Key);
```

Comment:
I don't think this causes any side effects since the InMemoryProvider was only used in the unit tests. But, this helped me to write a better test for my domain model object.